depop-app/src/hooks/useGetProducts.js

```diff
@@ -0,0 +1,29 @@
+import {useEffect, useState} from "react";
+
+export const useGetProducts = (url) => {
```

Comment:
I can't tell if this is too much to include or not, I'd be interested to know what others think as it is interesting to see different candidates approaches to data fetching